src/wallet/wallet.cpp

```diff
@@ -4258,6 +4305,71 @@ void CWallet::GetUnspentFilteredNotes(
                 throw std::runtime_error(strprintf("Error while decrypting note for payment address %s: %s", EncodePaymentAddress(pa), exc.what()));
             }
         }
+
```

Comment:
This is (almost?) duplicated code which should be factored out.

Reply:
Yep, but that can happen at some later point.